pkg/operator/ceph/file/mirror/config.go

```diff
@@ -22,23 +22,28 @@ import (
 	"github.com/rook/rook/pkg/daemon/ceph/client"
 	"github.com/rook/rook/pkg/operator/ceph/config"
 	"github.com/rook/rook/pkg/operator/ceph/config/keyring"
+	"github.com/rook/rook/pkg/operator/ceph/version"
 	"github.com/rook/rook/pkg/operator/k8sutil"
 )
 
 const (
 	keyringTemplate = `
 [client.fs-mirror]
 	key = %s
-	caps mon = "allow r"
+	caps mon = "allow profile cephfs-mirror"
 	caps mgr = "allow r"
 	caps mds = "allow r"
 	caps osd = "'allow rw tag cephfs metadata=*, allow r tag cephfs data=*'"
-
 `
 	user   = "client.fs-mirror"
 	userID = "fs-mirror"
 )
 
+var (
+	// PeerAdditionMinVersion This version includes a number of fixes for snapshots and mirror status
+	PeerAdditionMinVersion = version.CephVersion{Major: 16, Minor: 2, Extra: 2}
```

Comment:
After 16.2.5 is out, will we just make that the required min version for fs mirroring?

Reply:
Yes.